chitech_composite_processor/Utils_CombinedXSWriter.py

```diff
@@ -0,0 +1,71 @@
+"""Writes cross-sections to file using the combined Chi-tech data"""
+import numpy as np
+
+def WriteChiTechFile(data, chi_filename, comment = "# Output"):
```

Comment:
cannot use same name. combiner in the definition of the method.

to drive the point, change data to combined_data